### PR TITLE
Upkeep

### PR DESCRIPTION
**Pull request by dunkenwg**
I didn't really know who to put as a reviewer for this, so I decided to be safe and go with all three of you as this package is a CRAN package.

I'm going through our packages that use the set functions and making sure the code still works on the development version of R. You can see more info in this [issue](https://github.com/poissonconsulting/infrastructure-issues/issues/196).

Something that definitely requires your attention is commit [e0b9b32](https://github.com/poissonconsulting/dbflobr/pull/27/commits/e0b9b320cb2477596f381e3a0fa4fdfbfa12b7bd).

### CODE REVIEW

File: tests/testthat/test-check.R [hunk 1]
```diff
@@ -107,7 +107,7 @@ test_that("check_flob_query", {
   expect_identical(check_flob_query(slob_obj, slob = TRUE), slob_obj)
   expect_identical(check_flob_query(slob_obj, slob = NA), slob_obj)
 
-  expect_error(check_flob_query("non-blob", slob = NA), "`x` must be a blob of a serialized object.")
+  expect_error(check_flob_query("non-blob", slob = TRUE), "`x` must be a blob of a serialized object.")
```
Comment (joethorley) [inline tests/testthat/test-check.R:110]: What happens if `slob = NA`

**Review by joethorley — CHANGES_REQUESTED** [review]
looks great - that is a really good PR! I just have one question.

**Comment by dunkenwg** [comment]
@joethorley, @sebdalgarno, and @evanamiesgalonski, when you have a chance would you be able to review this pull request?

**Review by joethorley — APPROVED** [review]
looks good - there is a conflict that presumably we can resolve when merge?

**Comment by dunkenwg** [comment]
The conflict should be easy to resolve when we merge. It's just a difference in the README badges

**Review by evanamiesgalonski — APPROVED** [review]
Looks good to me. 
Once we've merged I'll do a pass through at some point and clean up some of the unit tests.

**Review by sebdalgarno — APPROVED** [review]
looks good thanks Duncan!

**Pull request merged**